test/test_endpoints.py

```diff
@@ -430,17 +430,20 @@ def test_endpoint_connect_tor(self, private_tor, global_tor, system_tor):
 class FakeProtocol(object):
     implements(IProtocol)
 
-    def dataReceived(self, data):
+    @staticmethod
+    def dataReceived(data):
         print "DATA", data
 
-    def connectionLost(self, reason):
+    @staticmethod
+    def connectionLost(reason):
         print "LOST", reason
```

Comment:
I'm also not sure why static methods are "a thing".  Not really sure why that's a thing.  Easy to turn that off

I'll broaden the coverage.